CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
 
+## Version 1.0.15
```

Comment:
Should this not be 1.0.16? Since the main POM is 1.0.16-SNAPSHOT now?
Also I suggest we make the next release 1.1.0 instead, since you have really added a new feature, not fixed a lot of bugs. If you agree I guess the pom version should be set to 1.1.0-SNAPSHOT?